media/visual.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="D:/Projects/Programming/callstackninja/media/visual.css">
+    <title>Stack Frame Visualization</title>
+</head>
+<body>
+
+<style>
+    html, body{
+    padding:0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: monospace;
+    }
+#container{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: #333;
+    width: 50%;
+    height: 90vh;
+    margin: 2.5% auto;
+    }
+
+.block-attrib{
+    width: 100%;
+    text-align: center;
+    border-bottom: 1px dashed black;
+    font-size: 1.2em;
+    color: #004AB5;
+}
+
+#stack{
+    border-top: 10px solid #2F4858;
+    border-right: 10px solid #2F4858;
+    border-left: 10px solid #2F4858;
+    transform:scaleY(-1);
+    overflow-x: auto; 
+    overflow-y: none;
+    height: 400px;
+    width: 70%;
+    margin: auto;
+}
+
+.title{
+    color: rgb(231, 100, 52);
+    border-bottom: 1px solid red;
+    font-size: 1.5em;
+    text-align: center;
+    padding: 2%;
+}
+
+.block{
+    background-color: #00C896;
+    width: 90%;
+    min-height: 15%;
+    margin: 1% auto;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;	
+    transform: scaleY(-1);
+}
+    .center{
+    text-align: center;
+    }
+    ::-webkit-scrollbar{
+    width: 6px;
+    }
+    
+    ::-webkit-scrollbar-thumb{
+    border-radius: 10px;
+    background-color: #eee;
+    height: 5px;
+    }
+    
+    ::-webkit-scrollbar-thumb:window-inactive{
+    background: rgba(100,100,100,0.4);
+    }
+    
+    .block-attrib:hover{
+    background-color: #06ebb2;
+    cursor: pointer;
+    }
+
+    .tree-node {
+    border: 1px solid #fff;
+    height: 200px;
+    width: 200px;
+    background-color: #242f38;
+    border-radius: 50%;
+    margin-bottom: 200px;
+    text-align: center;
+    vertical-align: middle;
+    }
+</style>
+
+<h2 class = "center">Stack Visualization (Stack View)</h2>
+<div id="stack-container">
+    <div id="stack"></div>
+</div>
+
+
+<h2 class = "center">Stack Visualization (Tree View)</h2>
+<div id="trees-container">
+</div>
+
+<sf id="sf-id" data-frames='{sfs}' ></sf>
+<script src="D:/Projects/Programming/callstackninja/media/visualize.js"></script>
```

Comment:
this path should be added as reference path not absolute. ../../xx.js

Reply:
It is solved using `vscode.Uri` instead of a reference path.